.github/workflows/lint.yml

```diff
@@ -1,6 +1,6 @@
 name: ❄️ Lint
 
-on: [pull_request]
+on: [push, pull_request]
```

Comment:
why do we need to change it in this PR?
one PR for one thing

Reply:
Thanks! Fix the first two unrelated changes. 
The last one is what this PR need, otherwise `make stop` could not revert the `config.yaml.bak`